src/Raven.Client/ServerWide/Operations/DeleteDatabasesOperation.cs

```diff
@@ -29,6 +29,21 @@ public DeleteDatabasesOperation(string databaseName, bool hardDelete, string fro
                 _parameters.FromNodes = new[] { fromNode };
         }
 
+        public DeleteDatabasesOperation(string databaseName, bool hardDelete, string fromNode, int shard, TimeSpan? timeToWaitForConfirmation = null)
```

Comment:
I wonder if we want a dedicated command for that...
`DeleteShardDatabasesOperation`?
cc @ppekrol